src/main/webapp/app/quiz/participate/multiple-choice-question/multiple-choice-question.component.ts

```diff
@@ -88,4 +96,17 @@ export class MultipleChoiceQuestionComponent implements OnInit, OnDestroy {
             }) !== -1
         );
     }
+
+    count(): void {
+        const answerOptions = this.question.answerOptions;
+        const selectedOptions = this.selectedAnswerOptions;
+
+        this.amountOfAnswerOptions = this.question.answerOptions.length;
+
+        this.correctAnswers = answerOptions.filter(option => option.isCorrect).length;
+        this.wrongAnswers = answerOptions.filter(option => !option.isCorrect).length;
+
+        this.chosenCorrectAnswerOption = selectedOptions.filter(option => option.isCorrect).length;
+        this.chosenWrongAnswerOption = selectedOptions.filter(option => !option.isCorrect).length;
+    }
```

Comment:
see the comment above. In my opinion these additional variable are not necessary and you can simply pass question and submittedAnswer to the other component.
Also I think you have a logic error here. There are 4 cases for MC:
1) Option is wrong and user has not chosen it --> correct choice
2) Option is wrong and user has chosen it --> wrong choice
3) Option is correct and user has not chosen it --> wrong choice
4) Option is correct and user has chosen it --> correct choice

This logic should be already available somewhere else and I think it would be great if you can reuse this logic instead of implementing it again (which would be duplicated code)

Reply:
I guess that the calculations are mentioned in QuizSubmissionService.ts meaning they are done in the backend. Am I correct? 

Also in the HTML file are functions which check how the result should be displayed - should there be a correct or wrong icon or a triangle warning sign - My thought was to adopt to those functions only in the frontend